assets/clickhouse/main.py

```
@@ -0,0 +1,162 @@
+
+from src.template import Benchmark, logger
+"""
+WORK_DIR: The in-container path to an accessible location e.g. "/home"
+Benchmark: Base class for benchmarks, has docker_execute to execute command within container
+logger: A logging.Logger
+"""
+
+CLICKHOUSE_COLLECTION_NAME = "clickhouse_bench"
+class clickhouse_native_json_bench(Benchmark):
+    # add any parameters to the tool here
+    def __init__(self, dataset, manual_column_names=True, keys=[], additional_order_by=[], timestamp_key=False):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)